README.md

```diff
@@ -169,7 +178,14 @@ Assuming the repository name is `repo-name`:
     - [ ] Add the release workflow to pypi.org's package publishing (and test.pypi.org's package publishing).
       Example can be found [here][pypi-publishing]
 
+### Release a new version
+
 - [ ] Have the maintainer push a new tag and walk them through the release process
+    - Find the workflow `Publish Python 🐍 distribution 📦 to PyPI` in the Actions tab.
```

Comment:
This name may not always be the same. I shortened it for drf-excel because they don't use test PyPI